python/runtime/utils/PyRemoteRESTQPU.cpp

```diff
@@ -89,7 +89,9 @@ class PyRemoteRESTQPU : public cudaq::BaseRemoteRESTQPU {
     pm.addPass(mlir::createCanonicalizerPass());
     pm.addPass(cudaq::opt::createApplyOpSpecializationPass());
     pm.addPass(createInlinerPass());
-    pm.addPass(cudaq::opt::createExpandMeasurementsPass());
+    /// NOTE: This change is for the `braket` target. May require additional
+    /// pass in other pipelines.
+    // pm.addPass(cudaq::opt::createExpandMeasurementsPass());
```

Comment:
Is this pipeline spec common to all targets?

Reply:
it is but I think it should be in the target.yml file already for cpp code to work

Reply:
I am leaning towards keeping this comment as reminder in case we see any failures in integration tests.